apps/site/pages/courses/[courseSlug]/index.tsx

```diff
@@ -270,19 +270,29 @@ function CourseHeader({
 						</ul>
 					</div>
 
-					{isEnrolled && (
-						<Link
-							href={`/courses/${course.slug}/${nextLessonSlug}`}
-							className="btn-primary"
-						>
-							<span>
-								{completion?.courseCompletion.completedLessonCount === 0
-									? "Starten"
-									: "Fortfahren"}
-							</span>
-							<PlayIcon className="h-5" />
-						</Link>
-					)}
+					{isEnrolled &&
+						((completion?.courseCompletion.completedLessonCount ?? 0) >=
+						content[0].content.length ? (
+							<Link
+								href={`/courses/${course.slug}/${content[0].content[0].slug}`}
+								className="btn-primary"
+							>
+								<span>Öffnen</span>
+								<PlayIcon className="h-5" />
+							</Link>
+						) : (
+							<Link
+								href={`/courses/${course.slug}/${nextLessonSlug}`}
+								className="btn-primary"
+							>
+								<span>
+									{(completion?.courseCompletion.completedLessonCount ?? 0) === 0
+										? "Starten"
+										: "Fortfahren"}
+								</span>
+								<PlayIcon className="h-5" />
+							</Link>
+						))}
```

Comment:
es kann schon in ordnung sein, dass mehrere Link elemente zurückgegeben werden. Da beide Link elemente allerdings beide das selbe CSS design mit dem gleichen PlayIcon haben, wirkt es eher wie ein duplicate. 

Folgendes könnte ein beispiel sein (wobei hier der tradeoff die hohe komplexität der Start/Öffnen/Fortfahren ist): 

```suggestion

					{isEnrolled && (
						<Link
							href={`/courses/${course.slug}/${
								nextLessonSlug ?? firstLessonFromChapter.slug
							}`}
							className="btn-primary"
						>
							<span>
								{nextLessonSlug
									? lessonCompletionCount === 0
										? "Starten"
										: "Fortfahren"
									: "Öffnen"}
							</span>
							<PlayIcon className="h-5" />
						</Link>
					)}

``` 

Ich empfinde die Lösung aber als mehr straight forward und gleichzeitig lesbarer